index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ethiopia Poetic Jazz</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/1652c6c81e.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="home" id="home">
+        <nav>
+            <a href="/index.html" class="logo"><span>Ethiopia Poetic Jazz</span></a>
+            <a href="#" class="open-menu"><i class="fas fa-bars"></i></a>
+            <ul class="menu">
+                <li><a href="/index.html" class="home-link">Home</a></li>
+                <li><a href="/about.html" class="about-link">About</a></li>
+                <li><a href="" class="close-menu"><i class="fas fa-times"></i></a></li>
+            </ul>
+        </nav>
+        <div class="home-container">
+            <img src="images/home-bg16.png" alt="">
```

Comment:
- [ ] OPTIONAL
It's a good practice to provide `alt` names for your images so as to improve the accessibility features of your site.  Kindly endeavor to add `alt` names to **all** your images.